vnext/templates/templateUtils.js

```diff
@@ -110,7 +110,12 @@ async function runNpmInstall(config = {}, options = {}) {
   }
 }
 
-async function updateProjectPackageJson(config = {}, options = {}, props = {}) {
+async function updateProjectPackageJson(
+  config = {},
+  options = {},
+  props = {},
+  saveOptions = true,
```

Comment:
Is there a case where `saveOptions` is false?

Reply:
Not yet, but it's a new behavior that we may want to disable for some templates, I'm not sure yet.